bitnami/spark/templates/secret.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "spark.fullname" . }}-secret
+  labels:
+    app.kubernetes.io/name: {{ include "spark.name" . }}
+    helm.sh/chart: {{ include "spark.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+type: Opaque
+data:
+{{- if eq (.Values.security.rpc.authenticationEnabled | quote ) "yes" }}
+  rpc-authentication-secret: {{ "RPCDefaultSecret" | b64enc | quote }}
+{{- end }}
+{{- if eq (.Values.security.ssl.enabled | quote ) "yes" }}
+  ssl-key-password: {{ "defaultSSLKeyPAssword" | b64enc | quote }}
```

Comment:
I see that we are not generating random strings. Is that expected?

https://github.com/helm/charts/blob/master/stable/wordpress/templates/secrets.yaml

Reply:
It was my mistake. Changed.